Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -193,7 +193,7 @@ def test_cellar_formula
   end
 
   def test_env
-    assert_match %r{CMAKE_PREFIX_PATH="#{HOMEBREW_PREFIX}[:"]},
+    assert_match /CMAKE_PREFIX_PATH="#{Regexp.escape(HOMEBREW_PREFIX)}[:"]/,
```

Comment:
This needs to be wrapped with `()`: http://bot.brew.sh/job/Brew%20Pull%20Requests/1111/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/readall___syntax/